src/widgets/helper/ScrollbarHighlight.hpp

```diff
@@ -6,17 +6,49 @@ class ScrollbarHighlight
 {
 public:
     enum Style : char { None, Default, Line };
-    enum Color : char { Highlight, Subscription };
+    enum Type { Highlight, Subscription, Invalid };
```

Comment:
should be possible to make this an `enum class`

Reply:
Resolved in 399ff7da.